Src/FluentAssertions/Configuration/TestFramework.cs

```diff
@@ -0,0 +1,14 @@
+namespace FluentAssertions.Configuration;
+
+/// <summary>
+/// The test frameworks supported by Fluent Assertions.
+/// </summary>
+public enum TestFramework
+{
+    XUnit2,
+    XUnit3,
+    TUnit,
+    MsTest,
```

Comment:
Suggestion: `MsTest` -> `MSTest`
This aligns it with the name of the [MSTest](https://www.nuget.org/profiles/MSTestFramework) nugets

Reply:
Technically that doesn't follow the .NET naming conventions (because of the three consecutive capitals), so I prefer to keep it as-is.